app/models/reducers/summary_statistics_reducer.rb

```diff
@@ -0,0 +1,175 @@
+module Reducers
+  class SummaryStatisticsReducer < Reducer
+    attr_reader :extracts
+
+    @@valid_operations = [
+      "count",
+      "min",
+      "max",
+      "sum",
+      "product",
+      "mean",
+      "sse",
+      "variance",
+      "stdev",
+      "first",
+      "median",
+      "mode"
+    ]
+
+    validate do
+      unless config["summarize_field"].present?
```

Comment:
Config field needs to be defined via DSL (see other reducers) so that the web UI will know how to render the edit form.

Reply:
done